src/context.rs

```diff
@@ -115,6 +115,16 @@ impl Context {
         }
     }
 
+    pub fn startup(&mut self, startup_type: TPM2_SU) -> Result<()> {
+        let ret = unsafe { Esys_Startup(self.mut_context(), startup_type) };
```

Comment:
There is a Startup type already defined in src/constants/types.rs so you do not need to use the ESYS type in the api.

Reply:
Ah, right. But this function is going to be troublesome with the tests anyway (given that swtpm already runs a startup, and doesn't like having it rerun, as a normal TPM should), so I may very well just drop this PR.

Reply:
We do actually have to run the CLI version of this command before running tests: https://github.com/parallaxsecond/rust-tss-esapi/blob/master/tests/all.sh#L18 . Plus, even without a test, the method might be useful on `Context`

Reply:
Yeah, I do not think it should be closed. I think this method needs to be available even without a test.